server/techtools.cpp

```diff
@@ -850,30 +832,26 @@ static void research_tech_lost(struct research *presearch, Tech_type_id tech)
   {
     // Check government.
     if (!can_change_to_government(pplayer, government_of_player(pplayer))) {
-      /* Lost the technology for the government; switch to random
-       * available government. */
-      struct government *pgov = pick_random_government(pplayer);
+      // Lost the technology for the government; switch to Anarchy.
+      struct government *pgov = game.government_during_revolution;
 
       notify_player(
           pplayer, nullptr, E_NEW_GOVERNMENT, ftc_server,
           _("The required technology for our government '%s' "
-            "was lost. The citizens have started a "
-            "revolution into '%s'."),
+            "was lost. The country has fallen in %s."),
```

Comment:
I think fallen "into" anarchy, is more grammatically correct ?  Why change the "into" to "in" ?